### ISSUE TITLE
FR: Condition before turning on

### ISSUE THREAD

**Issue by t0bse** [InventoCasa/ha-advanced-blueprints#35]
Hi,
it would be great, if i can set a condition before my device is turned on. 
i.E. my Airconditioner should only turn on, if it's hot enough. but this condition also needs a slider, to set a time, how often it should be checked again.

**Comment by eMerzh** [comment]
Not sure if i miss something, but agree here, 
like i have a dishwasher, it shouldn't run every day ... but if i could have a toggle saying "ready" then the automation would pick it up 🤔

**Comment by DAVIZINH0** [comment]
very interesting for my swiming pool and my pump.
I can create a entity "on" when the pool reach the maximun time i want the pump is working. (i use a counter for this)

**Comment by techydude** [comment]
Yes!  In the same way that standard Automations have 'conditions', it can be useful to have these to decide whether to enable a particular load.

**Comment by Maik7** [comment]
You can do that already. You can deactivate the automation if it is not hot enough and activate it, if it is hot again.

**Comment by techydude** [comment]
> You can do that already. You can deactivate the automation if it is not hot enough and activate it, if it is hot again.

Sure, but it adds an extra layer of complexity for the user to have to manage the automation using a second automation.  It 'standard' Automations is expected by users that Automations have Triggers, *and* Conditions.  This blueprint does not (yet) support Conditions.
